interfaces/acados_template/acados_template/c_templates_tera/multi_CMakeLists.in.txt

```diff
@@ -126,20 +126,20 @@ set(MODEL_SRC
     {{ model[jj].name }}_model/{{ model[jj].name }}_expl_ode_fun.c
     {{ model[jj].name }}_model/{{ model[jj].name }}_expl_vde_forw.c
     {{ model[jj].name }}_model/{{ model[jj].name }}_expl_vde_adj.c
-	{%- if hessian_approx == "EXACT" %}
+	{%- if solver_options.hessian_approx == "EXACT" %}
```

Comment:
There is some logic at the top of this file setting the option `hessian_approx`, which is an artifact from the single phase CMake file, which is also used when generating integrators.
This logic should be removed in this file and the multi Makefile.